app/Http/Controllers/API/v1/TrustedUserController.php

```diff
@@ -0,0 +1,108 @@
+<?php declare(strict_types=1);
+
+namespace App\Http\Controllers\API\v1;
+
+
+use App\Http\Resources\TrustedUserResource;
+use App\Models\TrustedUser;
+use App\Models\User;
+use Illuminate\Auth\Access\AuthorizationException;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+use Illuminate\Http\Response;
+
+class TrustedUserController extends Controller
+{
+
+    /**
+     * @OA\Get(
+     *     path="/user/{self|userId}/trusted",
+     *     operationId="trustedUserIndex",
+     *     summary="Get all trusted users for a user",
+     *     description="Get all trusted users for the current user or a specific user (admin only).",
+     *     tags={"User"},
+     *     @OA\Parameter(name="user", in="path", required=true, description="ID of the user (or string 'self' for current user)", @OA\Schema(type="integer")),
```

Comment:
<img width="465" alt="Screenshot 2024-07-05 at 18 44 17" src="https://github.com/Traewelling/traewelling/assets/1267894/eb4271d4-8a03-4725-98ba-b01b01c946c2">

You have to change the schema to a string or document two endpoints. 
Also the parameter has to be identical to the variable in the path.

so either:
```
path="/user/{userId}/trusted"
@OA\Parameter(name="userId"...)
```
or a longer docblock